templates/inspec/custom_functions/google_compute_region.rb

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# helper for returning a list of zone short names rather than fully qualified URLs e.g.
+#   https://www.googleapis.com/compute/v1/projects/spaterson-project/zones/asia-east1-a
+def zone_names
+  return [] if !exists?
+  @zones.map { |zone| zone.split('/').last }
+end
+
+def up?
+  return false if !exists?
+  @status == 'UP'
+end
+
+def to_s
+  name = @opts[:name] || @name
```

Comment:
I think this should be just `@name` as we no longer have `@opts` in MM resources. 

The constructor is passed `params` which should be identical to `opts` but it does not save a reference to it yet. That could be added if desired.

Reply:
`@name` is fine in all cases except for failure where we want to show e.g.
```
     ✔  Region notthere should not exist
```
instead of:
```
✔  Region  should not exist
```
`@opts` lives on via inheritance from `GcpResourceBase` so thought I'd leverage it here.

Reply:
Cool! Didn't know we could access `@opts` from here